bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+# Paper review-Perception Aware Model Predictive Control for Quadrotors 
+
+Authors - Davide Falanga, Philipp Foehn, Peng Lu, and Davide Scaramuzza
+
+## Why is this important?
```

Comment:
This section mention clearly about the importance of the proposed algorithm, although it could be better if the writer give more realistic example to emphasize the importance. (Realistic here means the actual case of application to give reader more understanding, especially those who are unfamiliar with autonomous vehicle)